photutils/detection/tests/test_peakfinder.py

```diff
@@ -148,7 +140,4 @@ def test_data_nans(self):
         """Test that data with NaNs does not issue Runtime warning."""
         data = np.copy(PEAKDATA)
         data[1, 1] = np.nan
-
-        with warnings.catch_warnings():
```

Comment:
This is no longer needed because we are turning all uncaught warnings into exceptions in `setup.cfg`.